documentation/modules/mtv-shared-disks.adoc

```diff
@@ -6,7 +6,7 @@
 [id="mtv-shared-disks_{context}"]
 = Migrating virtual machines with shared disks
 
-You can migrate VMware virtual machines (VMs) with shared disks by using the {project-first}. This functionality is available for cold migrations only. 
+You can migrate VMware virtual machines (VMs) with shared disks by using the {project-first}. This functionality is supported only for cold migrations and is not supported for shared boot disks. 
```

Comment:
I'm not sure about the word "supported only for cold migration". As this is not on our side as VMware does not allow enabling the Change Block Tracking (CBT), which is needed for the warm migration, on the shared disks.
I don't want to give the customer idea that we just don't support it but it could be done but we don't support it.

Reply:
@mnecas Is the original wording "available for cold migrations only" better?  same question for shared boot disks?

Reply:
@mnecas Please review this PR again. 
I changed "supported" to "available" in both places. Let me know if "is not supported" is technically more appropriate for shared boot disks.